src/chanjo2/endpoints/overview.py

```diff
@@ -0,0 +1,34 @@
+from os import path
+
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse
+from fastapi.templating import Jinja2Templates
+
+from chanjo2.demo import DEMO_OVERVIEW_QUERY_DATA
+from chanjo2.meta.handle_overview_content import get_overview_data
+from chanjo2.models.pydantic_models import GeneralReportQuery
+
+
+def get_templates_path() -> str:
+    """Returns the absolute path to the templates folder of this app."""
+    APP_ROOT: str = path.abspath(path.join(path.dirname(__file__), ".."))
```

Comment:
```suggestion
    APP_ROOT: str = path.abspath(path.join(path.dirname(__file__), ".."))
```
Hard to understand. Could it be split to make it easier to understand?

Would be great if possible to avoid using the private variable `__file__`.

What does ".." do?

Reply:
Not really, the __file__ is used to get the absolute path to the root of the project, and the ".." is used to access the single directories under it. Here you need to specify the absolute path to the "templates" folder. I don't know another way to do it. especially if the repo is nested like this.